trin-core/src/portalnet/overlay_service.rs

```diff
@@ -45,6 +46,8 @@ pub const FIND_CONTENT_MAX_NODES: usize = 32;
 /// With even distribution assumptions, 2**17 is enough to put each node (estimating 100k nodes,
 /// which is more than 10x the ethereum mainnet node count) into a unique bucket by the 17th bucket index.
 const EXPECTED_NON_EMPTY_BUCKETS: usize = 17;
+/// Bucket refresh lookup interval in seconds
+const BUCKET_REFRESH_INTERVAL: u64 = 60;
```

Comment:
Renamed. It is configurable with a constant for now :smile_cat:. Seriously, not sure if we need to add this to the CLI but I think it should not be part of this PR.

Reply:
Not sure if we have a use case to make it configurable right now but this line is in the diff because when changing the code in the previous version of the PR, I decided to extract it for better readability. I'll be more careful when renaming or making such unrelated changes in the future :innocent: .

This constant is about the overlay bucket refresh lookup logic which is out of the scope of this PR.